.buildkite/pipeline.yml

```diff
@@ -98,6 +98,21 @@ steps:
       - step: "tests"
         allow_failure: false
 
+  - label: ":argo: Update image tag for Fleet Server"
+    trigger: serverless-gitops-update-stack-image-tag
+    async: true
+    branches: main
+    build:
+      env:
+        IMAGE_TAG: "$DOCKER_IMAGE_TAG"
```

Comment:
I'm not sure if this is actually the right tag

Reply:
This environment variable does not exist in Buildkite or when the execution reaches this step.

What value should it be here ?